js-mentoring-program-master/tasks/strings/strings.js

```diff
@@ -51,7 +77,11 @@ function replace10(text) {
  * console.log(replaceConfidential("lorem [ipsum] si dolor")) // lorem [CONFIDENTIAL] si dolor
  */
 function replaceConfidential(text) {
+  const start = text.indexOf('[');
+  const end = text.indexOf(']');
 
+  const result = text.substr(0, start + 1) + 'CONFIDENTIAL' + text.substr(end, text.length);
```

Comment:
All the solution in one string: `return text.replace(regExp, replacement);` Your task is to come up with appropriate RegExp pattern and replacement string.  Use https://regex101.com/ for patterns testing